packages/twenty-front/src/modules/apollo/services/apollo.factory.ts

```diff
@@ -64,6 +69,7 @@ export class ApolloFactory<TCacheShape> implements ApolloManager<TCacheShape> {
             authorization: this.tokenPair?.accessToken.token
               ? `Bearer ${this.tokenPair?.accessToken.token}`
               : '',
+            'x-locale': this.currentWorkspaceMember?.locale || '',
```

Comment:
There are different ways to implement API localization. We could have automatically localized based on the user API's token, I tried it. From an implementation perspective implementing it this way was easier (workspaceMember object is difficult to access, locale should probably be moved to userWorkspace in the future). In the end I find this to be quite elegant and probably more suited (you can also decide to set a locale with an API key which wouldn't have been possible, and it gives flexibility to the frontend to decide when NOT to translate for example when editing a label)